packit_service/constants.py

```diff
@@ -67,6 +67,8 @@
 
 MSG_TABLE_HEADER_WITH_DETAILS = "| Name/Job | URL |\n" "| --- | --- |\n"
 
+REQUESTED_PULL_REQUEST_COMMENT = "/packit"
```

Comment:
Could we potentially bend this in future to be able to trigger only stage or only prod? :thinking: